core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -52,6 +54,7 @@ class CityInfo {
     var hasSoldBuildingThisTurn = false
     var isPuppet = false
 
+    var boughtPlaneThisRound = false
```

Comment:
It is too specific for the planes. Do we need the same for other types? I would not like to get much of these here.

Reply:
hmm thats a good question, right now we dont need this for any other type. Alternatively I could make a list of unittypes bought this round?